server/app/services/program/ProgramServiceImpl.java

```diff
@@ -1192,13 +1201,49 @@ private ProgramQuestionDefinition syncProgramQuestionDefinition(
     return pqd.loadCompletely(programDefinitionId, questionDefinition);
   }
 
-  /*
-   * Looks at the most recent version of each program and returns the program marked as the
-   * common intake form if it exists. The most recent version may be in the draft or active stage.
+  /**
+   * Clears the common intake form if it exists.
+   *
+   * <p>If there is a program among the most recent versions of all programs marked as the common
+   * intake form, this changes its ProgramType to DEFAULT, creating a new draft to do so if
+   * necessary.
    */
-  private Optional<ProgramDefinition> getCommonIntakeForm() {
-    return getActiveAndDraftPrograms().getMostRecentProgramDefinitions().stream()
-        .filter(p -> p.isCommonIntakeForm())
-        .findFirst();
+  private void clearCommonIntakeForm() {
+    Optional<ProgramDefinition> maybeCommonIntakeForm = getCommonIntakeForm();
+    ProgramDefinition draftCommonIntakeProgramDefinition =
+        programRepository
+            .createOrUpdateDraft(maybeCommonIntakeForm.get().toProgram())
+            .getProgramDefinition();
+    Program commonIntakeProgram =
+        draftCommonIntakeProgramDefinition.toBuilder()
+            .setProgramType(ProgramType.DEFAULT)
+            .build()
+            .toProgram();
+    programRepository.updateProgramSync(commonIntakeProgram);
+  }
+
+  private ImmutableSet.Builder<CiviFormError> validateProgramData(
```

Comment:
**question (blocking):** why return the builder not the built result?

Reply:
It was just to avoid having to copy all the elements in validateProgramDataForCreate but switched to returning the set since it's negligible performance-wise anyway.